Composer/packages/client/src/pages/botProject/create-publish-profile/PublishProfileDialog.tsx

```diff
@@ -165,7 +165,14 @@ export const PublishProfileDialog: React.FC<PublishProfileDialogProps> = (props)
 
       let arm, graph;
       if (!userShouldProvideTokens()) {
-        const tenantId = getTenantIdFromCache();
+        let tenantId = config.tenantId;
+
+        if (!tenantId) {
+          // eslint-disable-next-line no-console
+          console.log('Provision config does not include tenant id, using tenant id from cache.');
```

Comment:
should this get updated?

Reply:
This is just in case. This should not be executed anymore with this PR because I am passing in the tenant id when starting a provision.